spec/javascripts/Stats/usage_chart.spec.js

```diff
@@ -2,6 +2,15 @@ import { StatsUsageChart } from 'Stats/lib/usage_chart'
 
 describe('StatsUsageChart', () => {
   let chart = new StatsUsageChart({container: '#chart', groupedSeries: ['marvin', 'trillian', 'zaphod']})
+  const plotChart = {
+    load: jest.fn()
+  }
+  chart.plotChart = plotChart
+  const spy = jest.spyOn(plotChart, 'load')
```

Comment:
There's no need to both mock and spy on methods, you can simply use the reference to the mock:
```js
  const load = jest.fn()
  chart.plotChart = { load }
  
  ...
  expect(load).toHaveBeenCalledWith({ ... })
```